diff --git a/support/pma.c b/support/pma.c
index 6221e752..5c273819 100644
--- a/support/pma.c
+++ b/support/pma.c
@@ -345,7 +345,7 @@ static void flr(ao_t *p) {  // remove ao from free list
 #define MAP_NORESERVE 0
 #endif /* MAP_NORESERVE */
 
-#define MMAP(N) mmap(NULL, (N), PROT_NONE, MAP_PRIVATE | MAP_ANONYMOUS | MAP_NORESERVE, -1, 0)
+#define MMAP(N) mmap(NULL, (N), PROT_NONE, MAP_PRIVATE | MAP_ANON | MAP_NORESERVE, -1, 0)
 #define MUNMAP(A, N) do { if (0 != munmap((A), (N))) { ERR("munmap()" ERN); SERN; } } while (0)
 static void * addrgap(off_t n) {  // find big gap in address space to map n bytes
   void *A, *Amax = NULL;  size_t L, U, Max = 0, N = (size_t)n;  char *r;
@@ -702,4 +702,3 @@ void pma_set_avail_mem(const ul_t v) {
   }
   assert(!IC);
 }
-
